docs/building-apps.md

```diff
@@ -218,6 +219,25 @@ my-app/
   webos-meta/
 ```
 
+## Caching
+
+For supporting better [`Caching`](https://webpack.js.org/guides/caching/), Enact CLI provides `--content-hash` option to add a unique hash to each output file name based on the content of an asset.
+
+Building With this option should produce the following output:
+
+```none
+                       Asset       Size  Chunks                    Chunk Names
+main.7e2c49a622975ebd9b7e.js     544 kB       0  [emitted]  [big]  main
```

Comment:
Please provide an example with css files and image files too.

Reply:
OK, 
image files are not shown on bash, so I will update css only